### PR TITLE
Use our new uuidToObject() from plone.app.uuid.utils

### PR DESCRIPTION
**Pull request by anirudhhkashyap**
Fixes: #52 

As mentioned, I've used the new `uuidToObject()` from `plone.app.uuid.utils` wherever applicable.

### CODE REVIEW

File: plone/outputfilters/browser/resolveuid.py [hunk 1]
```diff
@@ -27,11 +28,8 @@ def uuidToURL(uuid):
     "But be aware that this does an extra security check."
 )
 def uuidToObject(uuid):
```
Comment (jensens) [inline plone/outputfilters/browser/resolveuid.py:30]: Please deprecate this function with a decorator -> https://docs.plone.org/develop/styleguide/deprecation.html#deprecating-methods-and-properties

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: plone/outputfilters/browser/resolveuid.py (hunk 1 above)
Comment (anirudhhkashyap) [reply plone/outputfilters/browser/resolveuid.py:27]: Have a look, are these the changes needed?

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]
LGTM, thanks.

Jenkins had some trouble, unrelated to this PR. Should be okay again, so let's try:

@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
Seems fine. Jenkins is broken for PRs, but I ran some tests locally and those were fine.
I will merge. Let's keep an eye on Jenkins to see if it stays/becomes green.
Thanks for your work!

**Comment by mauritsvanrees** [comment]
I have reverted this. It causes test failures in plone.outputfilters, plone.restapi and plone.volto. See [this Jenkins report](https://jenkins.plone.org/job/plone-6.0-python-3.9/lastCompletedBuild/testReport/). Please see if you can fix those.

**Pull request merged**